### DISCUSSION TITLE
Code coverage with jacoco

### DISCUSSION THREAD

**Discussion by paulroehr** [Banno/Gordon#126] [Q&A] [answered]
Hello Gordon Devs,

I'm currently using Gordon to be able to have more reliable tests. Especially to be able to rerun failed tests and let the tests run on more than one device. 

But I still have one problem: We are using jacoco to get our coverage report and it relies on the .ec coverage file, which is generated from the 'createDebugCoverageReport' gradle task. Unfortunately the task automatically starts the 'connectedAndroidTest' task, even though I have run all ui tests with gordon before. Is it possible to let Gordon create the coverage file without using the 'createDebugCoverageReport' task? 
Or is there another way to make Gordon compatible with jacoco?

Thanks in advance and Gordon rocks :D

**Comment by joshschriever** [comment, marked as answer]
See #91 and #114
